### ISSUE TITLE
Incorrect error when inserting `cpu,`

### ISSUE THREAD

**Issue by e-dard** [influxdb/influxdb#4770]
```
> insert cpu,
ERR: unable to parse 'cpu,': missing tag value
```

I would expect the error to be:

> ERR: unable to parse 'cpu,': missing tag **key**

I'm currently fixing #3070, depending on thoughts about this issue, I will address it at the same time.

**Comment by jwilder** [comment]
Yes.  Should say `missing tag key`.

**Comment by e-dard** [comment]
OK, great. I'll fix this too when along with #3070

**Comment by e-dard** [comment]
Resolved in #4841
